pages/forum.js

```diff
@@ -100,8 +99,11 @@ const ForumPage = ({ forumNote, query, appContext }) => {
             {authors.map((author) => (
               <meta key={author} name="citation_author" content={author} />
             ))}
-            <meta name="citation_publication_date" content={onlineDate} />
-            <meta name="citation_online_date" content={modificationDate} />
+            {onlineDate ? (
+              <meta name="citation_online_date" content={onlineDate} />
+            ) : (
+              <meta name="citation_publication_date" content={publicationDate} />
+            )}
```

Comment:
The logic here is basically: if the note has `odate` set use that for citation_online_date and don't include the citation_publication_date tag. Otherwise, show the citation_publication_date tag with the `pdate` or `cdate` of the note.

Reply:
I think there shouldn't be public note with no odate

Reply:
my only concern about removing the tag citation_publication_date is that some other tools may use it. For example: zotero (https://www.zotero.org/support/dev/exposing_metadata)

Looking at the documentation:

```
Date: citation_date (preferred), citation_publication_date, citation_cover_date, citation_online_date, citation_year
```

I think having citation_online_date only is enough.

Reply:
So all of the older venues are have odate? I thought some of them might be using pdate or not have either.

Reply:
I think @carlosmondra set the odate for all of them.